LICENSE

```diff
@@ -1,22 +1,688 @@
+This file includes licensing information for Arduino GitHub Actions.
 
-The MIT License (MIT)
+Copyright (c) 2019
```

Comment:
Here is missing the name of the copyright holder I think it should be
`Copyright (c) 2019 ARDUINO SA`